server/services/prison/PrisonRegisterService.ts

```diff
@@ -1,29 +1,37 @@
-import RestClient from '../../data/restClient'
 import config from '../../config'
+import PrisonRegisterStore from '../../data/cache/PrisonRegisterStore'
+import RestClient from '../../data/restClient'
 import { PrisonDto } from '../../@types/prisonRegisterApiClientTypes'
-
-export interface Prison {
```

Comment:
This has been moved into a source file of its own (see below), because otherwise it was introducing a circular dependency (ie. `PrisonRegisterService` depends on `PrisonRegisterStore`, but `PrisonRegisterStore` depended on `PrisonRegisterService` to get the `Prison` type)